src/obfl-specification.html

```diff
@@ -1350,7 +1352,7 @@ <h4 id="L8491" class="include">style</h4>
 <div class="sidebar markup">
 <dl>
   <dt>Usage</dt>
-    <dd>before, after, toc-entry, block, item, span, style</dd>
+    <dd>before, after, toc-entry, block, toc-block, item, span, style</dd>
```

Comment:
As ```toc-block``` is used for nesting and text is not allowed then ```style``` is not required or needed.